databases/week1/homework1.sql

```diff
@@ -0,0 +1,62 @@
+-- 1. Select the names and phones of all users;
+
+SELECT id, phone FROM user;
```

Comment:
Since you need to get names and phones, it would probably be better to get `name` instead of `id` ☺️

Reply:
Right :)